### ISSUE TITLE
remove get_stacktrace / make cowboy debuggable under OTP 23 / runnable unto OTP 24

### ISSUE THREAD

**Issue by RoadRunnr** [ninenines/cowboy#1469]
get_stacktrace/0 will be remove in OTP 24, but its use is already a problem for debugging in OTP 23.
Starting with OTP 23 get_stacktrace/0 always returns `[]`, this leads to very uninformative error messages like

```
2020-09-27T14:54:13.849015+00:00 <0.16052.0> error: Ranch listener ergw_http_listener, connection process <0.16052.0>, stream 1 had its request process <0.16053.0> exit with reason function_clause and stacktrace []                                                                                                                                                                                                        
2020-09-27T14:54:43.852343+00:00 <0.16069.0> error: crasher: initial call: cowboy_stream_h:request_process/3, pid: <0.16069.0>, registered_name: [], error: {function_clause,[]}, ancestors: [<0.16068.0>,<0.1207.0>,<0.1206.0>,ranch_sup,<0.1137.0>], message_queue_len: 0, messages: [], links: [<0.16068.0>,#Port<0.1192>], dictionary: [], trap_exit: false, status: running, heap_size: 4185, stack_size: 28, reductions: 2986352; neighbours:                                                                                                                                                                                          
```

**Comment by essen** [comment]
It was removed in 3a7232b019f975a594a696eace46abcbfeec5b2e.

**Comment by RoadRunnr** [comment]
argh, sorry for the noise. I thought I was locking at master, but was really still using 2.7.

**Comment by essen** [comment]
Oh I actually released a version 2.8 that contains the fix! I thought I didn't. So the situation is not as bad as I was thinking.
